tests/unit/executors/encoders/test_numeric.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import pytest
+
+from jina.executors.encoders.numeric import TransformEncoder
+
+input_dim = 5
+target_output_dim = 5
+
+
+class SimpleModel:
+    def fit(self, data):
+        return data
+
+    def transform(self, data):
+        return data
+
+
+simple_model = SimpleModel()
+encoder = TransformEncoder(output_dim=target_output_dim)
+
+
+def test_transformencoder_train():
+    train_data = np.random.rand(100, input_dim)
+    encoder.model = simple_model
+    encoder.train(train_data)
+    assert encoder.train
```

Comment:
what is exactly being asserted here?

Reply:
A typo. will be fixed in the next commit. Intended to `assert encoder.is_trained`